src/coreComponents/physicsSolvers/wavePropagation/sem/elastic/secondOrderEqn/isotropic/ElasticWaveEquationSEMKernel.hpp

```diff
@@ -199,7 +199,7 @@ class ExplicitElasticSEMBase : public finiteElement::KernelBase< SUBREGION_TYPE,
                               StackVariables & stack ) const
   {
 
-    m_finiteElementSpace.template computeFirstOrderStiffnessTerm( q, stack.xLocal, [&] ( int i, int j, real64 val, real64 J[3][3], int p, int r )
+    m_finiteElementSpace.computeFirstOrderStiffnessTerm( q, stack.xLocal, [&] ( int i, int j, real64 val, real64 J[3][3], int p, int r )
```

Comment:
those parameters can also be `const` I think.

Reply:
...and perhaps the `J[3][3]` needs to be passed as a reference.